app/controllers/admin/AdminController.php

```diff
@@ -155,9 +155,14 @@ public function applicationsShow($id)
     // would significantly reduce the number of SQL queries needed, as well
     // as the number of variables passed to the view.
     $application = Application::getUserApplication($id);
-    $profile = Profile::getUserProfile($id);
+    //only load the user profile if it has been filled out
+    if (Profile::where('user_id', '=', $id)->exists()) {
+      $profile = Profile::getUserProfile($id);
+    }
```

Comment:
i think this should always exist, because they can't get past the first step without a profile